test/e2e/util/fixture.go

```diff
@@ -259,9 +261,7 @@ func destroyCluster(ctx context.Context, t *testing.T, hc *hyperv1.HostedCluster
 		return azure.DestroyCluster(ctx, opts)
 	case hyperv1.PowerVSPlatform:
 		opts.PowerVSPlatform = core.PowerVSPlatformDestroyOptions{
-			BaseDomain:    hc.Spec.DNS.BaseDomain,
-			CISCRN:        hc.Spec.Platform.PowerVS.CISInstanceCRN,
-			CISDomainID:   hc.Spec.DNS.PrivateZoneID,
```

Comment:
how are these fields derived in the delete logic then? if these values aren't used anymore then can we just remove from the `PowerVSPlatformDestroyOptions` struct itself?

Reply:
https://github.com/openshift/hypershift/blob/main/cmd/cluster/powervs/destroy.go#L71 here we are using, when HostedCluster is not nil, we can get these values from that.
Using this to avoid some network calls again to get these details from BaseDomain like here https://github.com/openshift/hypershift/blob/main/cmd/infra/powervs/destroy.go#L246.

Here also we can add these two values after validating HostedCluster is not nil, but in this whole block, HostedCluster is not used to access the values. So removed them.